baremaps-cli/src/main/java/org/apache/baremaps/cli/iploc/Serve.java

```diff
@@ -42,7 +42,7 @@ public class Serve implements Callable<Integer> {
   private String host = "localhost";
 
   @Option(names = {"--port"}, paramLabel = "PORT", description = "The port of the server.")
-  private int port = 9000;
+  private int port = 3000;
```

Comment:
Why is the port change needed? I would use the same default port everywhere.

Reply:
I have put back the default to 9000 everywhere.